stock_indicators/indicators/marubozu.py

```diff
@@ -0,0 +1,48 @@
+from typing import Iterable, TypeVar
+
+from stock_indicators._cslib import CsIndicator
+from stock_indicators._cstypes import List as CsList
+from stock_indicators.indicators.common.candles import CandleResult, CondenseMixin
+from stock_indicators.indicators.common.results import IndicatorResults
+from stock_indicators.indicators.common.quote import Quote
+
+
+def get_marubozu(quotes: Iterable[Quote], min_body_percent: float = 0.95):
+    """Get Marubozu calculated.
+
+    (preview)
+    Marubozu is a single candlestick pattern that has no wicks,
+    representing consistent directional movement.
+
+    Parameters:
+        `quotes` : Iterable[Quote]
+            Historical price quotes.
+
+        `min_body_percent` : float, defaults 0.95
+            Minimum candle body size as decimalized percentage.
+
+    Returns:
+        `MarubozuResults[MarubozuResult]`
+            MarubozuResults is list of MarubozuResult with providing useful helper methods.
+
+    See more:
+         - [Marubozu Reference](https://daveskender.github.io/Stock.Indicators.Python/indicators/Marubozu/#content)
+         - [Helper Methods](https://daveskender.github.io/Stock.Indicators.Python/utilities/#content)
+    """
+    results = CsIndicator.GetMarubozu[Quote](CsList(Quote, quotes), min_body_percent)
+    return MarubozuResults(results, MarubozuResult)
+
+
+class MarubozuResult(CandleResult):
```

Comment:
Is it intentional that the name of result class for Marubozu is `CandleResult`? I just make it as `MarubozuResult` for consistency.

Reply:
Candlestick patterns are generally in Preview status, so this is subject to change if it poses a problem.

Reply:
I've updated that now Candlestick patterns, like Marubozu, use CandleResults.